content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -135,6 +148,102 @@ $ kubectl exec deploy/notsleep -- curl -s http://productpage:9080/ | grep -o "<t
 <title>Simple Bookstore App</title>
 {{< /text >}}
 
+{{< /tab >}}
+
+{{< tab name="Gateway API" category-value="gateway-api" >}}
+
+Create a [Kubernetes Gateway](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io%2fv1beta1.Gateway),
+so you can access the bookinfo app from outside the cluster:
+
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: bookinfo-gateway
+  namespace: istio-system
+spec:
+  gatewayClassName: istio
+  listeners:
+  - name: http
+    port: 80
+    protocol: HTTP
+    allowedRoutes:
+      namespaces:
+        from: All
+EOF
+{{< /text >}}
+
+The Gateway is configured to listen for HTTP traffic on port 80 and allow
+[HTTPRoutes](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io%2fv1beta1.HTTPRoute) from any
+namespace to bind to the listener.
+
+Creating a Kubernetes Gateway resource deploys an associated
+[proxy service](/docs/tasks/traffic-management/ingress/gateway-api/#automated-deployment), so run the following command
+to wait for the gateway to be ready.
+
+If you are using a cluster that supports a [load balancer Service](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer),
+wait until the Gateway is ready:
+
+{{< text bash >}}
+$ kubectl wait --for=condition=ready gtw/bookinfo-gateway -n istio-system
+{{< /text >}}
+
+Otherwise, wait until the Gateway deployment is available:
```

Comment:
Gateway API doesn't really work on a cluster without external LB.

Reply:
See https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports

Reply:
I see. I think that since this is still in flux, and because we will need to change a bunch of docs for this, I would leave it out of here for now and we can update it, along with all the other docs, in a followup PR. WDYT?

Reply:
> See https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports

The ambient doc currently uses a standard kind cluster (no MetalLB) and due to istio/istio#44035, the Gateway only surfaces the load balancer address in status. Instead of bringing in MetalLB as a dependency of the doc, I will update the PR to wait for the Gateway deployment.

Reply:
> I would leave it out of here for now and we can update it, along with all the other docs, in a followup PR. WDYT

IMO it's still a good idea to wait for the Gateway deployment to be ready (for those who are quick copy/paste'ers ;-))

Reply:
> IMO it's still a good idea to wait for the Gateway deployment to be ready ...

But IIUC, @howardjohn has removed the ready condition from the gateway status now, so every gateway will never be ready.

Reply:
We can use Programmed

Reply:
>But IIUC, @howardjohn has removed the ready condition from the gateway status now, so every gateway will never be ready.

Yes, but the Gateway's deployment will be ready and that's the `wait` that I was referring to.

Reply:
I updated the doc to check for the gtw's "Programmed=true" condition.